app/MMB/src/scenes/timeline/events/BookedFlightTimelineEvent.js

```diff
@@ -1,42 +1,21 @@
 // @flow
 
 import * as React from 'react';
-import idx from 'idx';
-import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
-import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
+import { Translation } from '@kiwicom/mobile-localization';
 
-import type { BookedFlightTimelineEvent as BookedFlightTimelineEventType } from './__generated__/BookedFlightTimelineEvent.graphql';
+import type { TimelineEvent as TimelineEventType } from '../__generated__/TimelineQuery.graphql';
```

Comment:
Are you sure `TimelineEvent` type exists? I wasn't able to find it. Also, you can see from this diff the prop-drilling (`data` are not used - only passed down). That looks weird since we have fragments and every component can fetch what's needed in that component. (not fetching for the other components)

Reply:
Oh right, so I should use a Relay Fragment Container on the `TimelineEvent` component, since that's where I handle the timestamp. And remove it from there. That makes sense. Thank you!